config/clowder_config.rb

```diff
@@ -31,12 +35,18 @@ def self.instance
         options["databaseName"] = config.database.name
         options["databaseUsername"] = config.database.username
         options["databasePassword"] = config.database.password
+        options["sslMode"] = config.database.sslMode
```

Comment:
@hsong-rh Do we need this DB parameter if everything related to DB is going to be set in the entrypoint script?